docs/api.md

```diff
@@ -671,12 +671,51 @@ const result = await page.evaluate(() => {
 console.log(result); // prints "56"
 ```
 
+Passing arguments to pageFunction.
+
+```js
+const name = 'Thomas';
```

Comment:
Let's amend the example on line 668 instead so that it accepts an argument:
```js
const result = await page.evaluate(x => {
  return Promise.resolve(8 * x);
}, 7);
console.log(result); // prints "56"
```
 
This should cover this and the following examples.